### PR TITLE
porting v5 init

### PR DESCRIPTION
**Pull request by panquez**

### CODE REVIEW

File: src/geode/io/model/svg_input.cpp [hunk 1]
```diff
@@ -36,6 +36,7 @@
 
 #include <geode/mesh/builder/edged_curve_builder.h>
 #include <geode/mesh/builder/point_set_builder.h>
+#include <geode/mesh/core/edged_curve.h>
```
Comment (BotellaA) [inline src/geode/io/model/svg_input.cpp:39]: why?

File: src/geode/io/mesh/ply_output.cpp [hunk 2]
```diff
@@ -34,7 +34,8 @@ namespace geode
         void PLYOutput::write() const
         {
             detail::AssimpMeshOutput< PolygonalSurface3D > impl{ filename(),
-                polygonal_surface(), "plyb" };
+                // polygonal_surface(), "plyb" };
```
Comment (BotellaA) [inline src/geode/io/mesh/ply_output.cpp:37]: To remove?

File: src/geode/io/model/svg_input.cpp (hunk 1 above)
Comment (panquez) [reply src/geode/io/model/svg_input.cpp:39]: It is needed for compilation, but I think previously it was included in another included header file (<geode/mesh/builder/edged_curve_builder.h>?), and not anymore

Comment (panquez) [reply src/geode/io/model/svg_input.cpp:39]: or geode/model/mixin/line.h ?

**Pull request merged**